GraphAlgorithms/Program.cs

```diff
@@ -27,10 +28,35 @@ private static void Main(string[] args)
         }, nameOption, algorithmTypeOption);
         rootCommand.Add(dimacsCommand);
 
-        var cliqueCommand = new Command("clique", "Find clique");
+        var cliqueCommand = new Command("max-clique", "Find maximum clique");
+        var fileOption = new Option<string>(name: "--file",
```

Comment:
maybe add options to command right after creating it or even cliqueCommand.Add(new Options...)
Tight now its difficult to make usre if all options are added